themes/doks/layouts/partials/header/header.html

```diff
@@ -36,20 +40,38 @@
       {{ end -}}
 
       <a class="navbar-brand order-1" href="{{ .Site.BaseURL | absURL }}" aria-label="{{ .Site.Title }} home page">
-        <img src="{{ "images/r3.svg" | absURL }}" alt="R3 Docs" class="d-lg-none ms-2 me-3" height="32">
-        <img src="{{ "images/r3.svg" | absURL }}" alt="R3 Docs" class="d-none d-lg-inline-block me-4"  height="40">
+        <img src="{{ " images/r3.svg" | absURL }}" alt="R3 Docs" class="d-lg-none ms-2 me-3" height="32">
+        <img src="{{ " images/r3.svg" | absURL }}" alt="R3 Docs" class="d-none d-lg-inline-block me-4" height="40">
```

Comment:
@zhengmar3 was this space intentional?
Logo is not displaying correctly now on the site and it seems this could be the cause?
![image](https://user-images.githubusercontent.com/103633799/225771860-1c785a6b-eccd-4e37-8234-0c143ef347cd.png)